contracts/SecurityModule.sol

```diff
@@ -62,13 +58,37 @@ contract SecurityModule is BaseModule, Initializable {
         IWallet(_wallet).authoriseModule(_module, true, data);
     }
 
+    function setLockedSecurityPeriod(address _wallet, uint lockedSecurityPeriod) public onlyOwner(_wallet) {
+        SignerConfInfo storage signerConfInfo = signerConfInfos[_wallet];
```

Comment:
if we just read the signerConfInfos, using memory to replace storage is better regarding the gas cost.